SimPEG/regularization/__init__.py

```diff
@@ -169,87 +168,3 @@
     AmplitudeSmallness,
     AmplitudeSmoothnessFirstOrder,
 )
-
-
-@deprecate_class(removal_version="0.19.0", future_warn=True)
-class SimpleSmall(Smallness):
-    """Deprecated class, replaced by Smallness."""
-
-    pass
-
-
-@deprecate_class(removal_version="0.19.0", future_warn=True)
-class SimpleSmoothDeriv(SmoothnessFirstOrder):
-    """Deprecated class, replaced by SmoothnessFirstOrder."""
-
-    pass
-
-
-@deprecate_class(removal_version="0.19.0", future_warn=True)
-class Simple(WeightedLeastSquares):
-    """Deprecated class, replaced by WeightedLeastSquares."""
-
-    def __init__(self, mesh=None, alpha_x=1.0, alpha_y=1.0, alpha_z=1.0, **kwargs):
-        # These alphas are now refered to as length_scalse in the
-        # new WeightedLeastSquares regularization
-        super().__init__(
-            mesh=mesh,
-            length_scale_x=alpha_x,
-            length_scale_y=alpha_y,
-            length_scale_z=alpha_z,
-            **kwargs
-        )
-
-
-@deprecate_class(removal_version="0.19.0", future_warn=True)
-class Tikhonov(WeightedLeastSquares):
-    """Deprecated class, replaced by WeightedLeastSquares."""
-
-    def __init__(
-        self, mesh=None, alpha_s=1e-6, alpha_x=1.0, alpha_y=1.0, alpha_z=1.0, **kwargs
-    ):
-        super().__init__(
-            mesh=mesh,
-            alpha_s=alpha_s,
-            alpha_x=alpha_x,
-            alpha_y=alpha_y,
-            alpha_z=alpha_z,
-            **kwargs
-        )
-
-
-@deprecate_class(removal_version="0.19.0", future_warn=True)
-class Small(Smallness):
-    """Deprecated class, replaced by Smallness."""
-
-    pass
-
-
-@deprecate_class(removal_version="0.19.0", future_warn=True)
-class SmoothDeriv(SmoothnessFirstOrder):
-    """Deprecated class, replaced by SmoothnessFirstOrder."""
-
-    pass
-
-
-@deprecate_class(removal_version="0.19.0", future_warn=True)
-class SmoothDeriv2(SmoothnessSecondOrder):
-    """Deprecated class, replaced by SmoothnessSecondOrder."""
-
-    pass
-
-
-@deprecate_class(removal_version="0.19.0", future_warn=True)
-class PGIwithNonlinearRelationshipsSmallness(PGIsmallness):
-    """Deprecated class, replaced by PGIsmallness."""
-
-    def __init__(self, gmm, **kwargs):
-        super().__init__(gmm, non_linear_relationships=True, **kwargs)
-
-
-@deprecate_class(removal_version="0.19.0", future_warn=True)
-class PGIwithRelationships(PGI):
-    """Deprecated class, replaced by PGI."""
-
-    def __init__(self, mesh, gmmref, **kwargs):
-        super().__init__(mesh, gmmref, non_linear_relationships=True, **kwargs)
```

Comment:
Sounds good. I thought we just wanted to remove them, but since we already have the machinery to raise an error, it sounds good. We also give users more time to adjust their codes.